modules/installation-network-user-infra.adoc

```diff
@@ -131,15 +131,26 @@ the Cluster Version Operator on port `9099`.
 |Port
 |Description
 
-.2+|TCP
-|`2379`-`2380`
-|etcd server, peer, and metrics ports
-
+|TCP
 |`6443`
 |Kubernetes API
 
 |===
 
+.Control plane machines to control plane machines
+[cols="2a,2a,5a",options="header"]
+|===
+
+|Protocol
+|Port
+|Description
+
+|TCP
+|`2379`-`2380`
+|etcd server, peer, and metrics ports
```

Comment:
please drop metrics that is now `9979` covered in separate section.

Reply:
otherwise LGTM

Reply:
Done, thanks